<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExtractWindow</class>
 <widget class="QDialog" name="ExtractWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>260</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_confirm">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>260</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Extract data</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_from">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>391</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_to">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>391</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_change_path">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>70</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Change</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>From:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>To:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string> Unzipping algorithm:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_lz77">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>LZ77</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_huffman">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Huffman</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
